snorkel/labeling/analysis.py

```diff
@@ -11,87 +11,123 @@
 Matrix = Union[np.ndarray, sparse.csr_matrix]
 
 
-############################################################
-# Label Matrix Diagnostics
-############################################################
 def _covered_data_points(L: Matrix) -> np.ndarray:
-    """Returns an indicator vector where ith element = 1 if x_i is labeled by at
-    least one LF."""
+    """Get indicator vector z where z_i = 1 if x_i is labeled by at least one LF."""
     return np.ravel(np.where(L.sum(axis=1) != 0, 1, 0))
 
 
 def _overlapped_data_points(L: Matrix) -> np.ndarray:
-    """Returns an indicator vector where ith element = 1 if x_i is labeled by
-    more than one LF."""
+    """Get indicator vector z where z_i = 1 if x_i is labeled by more than one LF."""
     return np.where(np.ravel((L != 0).sum(axis=1)) > 1, 1, 0)
 
 
 def _conflicted_data_points(L: sparse.spmatrix) -> np.ndarray:
-    """Returns an indicator vector where ith element = 1 if x_i is labeled by
-    at least two LFs that give it disagreeing labels."""
+    """Get indicator vector z where z_i = 1 if x_i is labeled differently by two LFs."""
     m = sparse.diags(np.ravel(L.max(axis=1).todense()))
     return np.ravel(np.max(m @ (L != 0) != L, axis=1).astype(int).todense())
 
 
 def label_coverage(L: Matrix) -> float:
-    """Returns the **fraction of data points with > 0 (non-zero) labels**
-    Args:
-        L: an n x m scipy.sparse matrix where L_{i,j} is the label given by the
-            jth LF to the ith item
+    """Compute the fraction of data points with at least one label.
+
+    Parameters
+    ----------
+    L
+        Matrix where L_{i,j} is the label given by the jth LF to the ith candidate
+
+    Returns
+    -------
+    float
+        Fraction of data points with labels
```

Comment:
I think adding quick examples for all of these good since very user facing- should make sure users get metrics